R/01-webdata-obj.R

```diff
@@ -1,7 +1,42 @@
 pkg.env <- new.env()
+pkg.env$gconfig <- list('wps.url'='http://cida.usgs.gov/gdp/process/WebProcessingService',
+                        'sleep.time' = 5, 
+                        'wait' = FALSE,
+                        'email' = as.character(NA),
+                        'verbose' = FALSE)
 .onLoad <- function(libname, pkgname){
   setJobState()
 }
+
+
+gconfig <- function(..., no.readonly = FALSE){
+  stopifnot(!no.readonly)
+  .gconfig.readonly <- c() # nothing right now?
+  single <- FALSE
+  args <- list(...)
+  if (!length(args)) 
```

Comment:
i like to be explicit about logical tests like these, e.g. `length(args) == 0`. mostly for readability (i forget whether 0 or non-0 is TRUE), and because long ago a prof told me to.